examples/meg_lasso_vs_wlassos.py

```diff
@@ -0,0 +1,86 @@
+import mne
```

Comment:
can you not merge this file with the plot_meg one ?

Reply:
@QB3 what do we do with this file, delete or it's needed ? It will not be run by Circle as its name does not start with `plot_*`